internal/platform/platform.go

```diff
@@ -73,6 +75,12 @@ func detect() Platform {
 		HostInfo: info,
 	}
 	p.detectPlatform()
+	if hasGpu, err := accelerators.HasNvidiaGpu(); err != nil {
```

Comment:
If you put this in `detectPlatform` in `platform_unix.go` instead then you don't need `nvidia_gpu_windows.go` at all

Reply:
Done!